site/pages/react/authentication-methods.mdx

```diff
@@ -12,6 +12,51 @@ In this guide, we'll show you two different ways to authenticate users:
 1. Using our UI components to handle the entire authentication flow.
```

Comment:
When we do the stuff about breaking up the pages in the previous comment let's make sure to link 1 and 2 to those pages!